devspaces-samples/ansible-devspaces-demo
# Enable Ansible Lightspeed

Add .code-workspace file with settings to enable Ansible Lightspeed automatically.

![screenshot-nimbusweb me-2024 07 23-12_51_21](https://github.com/user-attachments/assets/1278850d-e405-42e4-9cd9-1d64ddd21850)


**Related issue:** https://issues.redhat.com/browse/CRW-6336

**How to test:**
1/ Create a workspace using
[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://devspaces.apps.sandbox-stage.gb17.p1.openshiftapps.com/dashboard/#/https://github.com/devspaces-samples/ansible-devspaces-demo/tree/sv-enable-lightspeed)
2/ When the workspace is started and Ansible extension is initialized, Ansible Lightspeed should be enabled and it should be ready to connect to the server 

It's not possible to do this via .vscode/settings.json file because Ansible extension doesn't apply `ansible.lightspeed.enabled` settings that was set on Workspace Folder level. Here is the corresponding issue: https://github.com/ansible/vscode-ansible/issues/1443
